src/activities/Elsa.Activities.SQL/Activities/SqlQuery/ExecuteSqlQuery.cs

```diff
@@ -19,15 +26,15 @@ namespace Elsa.Activities.Sql.Activities
         Description = "Execute given SQL query and returned execution result",
         Outcomes = new[] { OutcomeNames.Done }
     )]
-    public class ExecuteSqlQuery : Activity
+    public class ExecuteSqlQuery : Activity, IActivityPropertyOptionsProvider, IRuntimeSelectListProvider
     {
         /// <summary>
         /// Allowed databases to run SQL
         /// </summary>
         [ActivityInput(
             UIHint = ActivityInputUIHints.Dropdown,
             Hint = "Allowed databases to run SQL.",
-            Options = new[] { "MSSQL Server", "PostgreSql" },
+            Options = new[] { "", "MSSQLServer", "PostgreSql" },
```

Comment:
Is this still required? I submitted a fix in Elsa 2.6 to make the first option selected by default.